### PR TITLE
Updates to use pgx 0.5.3

### PR DESCRIPTION
**Pull request by BradyBonnette**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -29,8 +29,8 @@ verify = ["dep:geiger"]
 [dependencies]
 current_platform = "0.2.0"
 cfg-if = "1"
-pgx = { version = "0.5.0-beta.1" }
```
Comment (thomcc) [inline Cargo.toml:32]: Nit: `~` doesn't make a difference here from the default (and anything aside from the default hits the "cargo's resolver behaves badly" behavior)

Comment (thomcc) [reply Cargo.toml:32]: Oh, unless this is a hack to make patch.crates-io work? I don't *think* it should matter for that but... maybe?

Comment (workingjubilee) [reply Cargo.toml:32]: I was mostly experimenting, tbh.

**Comment by workingjubilee** [comment]
Ah yep that'll do it. Okay, just need to get to 0.5.3 then.

**Review by workingjubilee — APPROVED** [review]
Feel free to merge at your leisure, @BradyBonnette.

**Pull request merged**